i18n/en.json

```diff
@@ -367,6 +367,14 @@
     "id": "api.channel.rename_channel.cant_rename_group_messages.app_error",
     "translation": "You cannot rename a group message channel"
   },
+  {
+    "id": "api.channel.undelete_channel.unarchived",
+    "translation": "{{.Username}} unarchived the channel."
+  },
+  {
+    "id": "api.channel.undelete_channel.undeleted.app_error",
+    "translation": "Unable to unarchived a not archived channel"
```

Comment:
I'll defer to PM on correct text messages introduced here.

Reply:
Thank you Saturn for the ping, I missed this during initial review.

@allenlai18 confirming if this error message only occurs when you are trying to restore/unarchive a channel that wasn't previously archived? Or are there any other cases where this error may be shown?

Reply:
Yes. that is the case when this error happens